lagom-client-restaurant/menu-item-impl/src/main/resources/application.prod.conf

```diff
@@ -2,63 +2,18 @@ include "application.conf"
 
 play.http.secret.key = "unused secret"
 
-http {
-  address = ${?HTTP_BIND_ADDRESS}
-  port = 9000
-}
-
-play.filters.hosts {
-  # Requests that are not from one of these hosts will be rejected.
-  allowed = [${?ALLOWED_HOST}]
-}
-akka.discovery.method = kubernetes-api
+# Workaround for https://github.com/lagom/lagom/issues/2239
+akka.discovery.method = akka-dns
 
 akka.management.cluster.bootstrap {
   contact-point-discovery {
     discovery-method = kubernetes-api
     service-name = "menu-item"
-    port-name = "akka-mgmt-http"
```

Comment:
It's no longer required to specify this explicitly. If you don't, each node will assume that the port number it binds to for Akka Management HTTP is the same port number used by other members of the cluster, and use that for bootstrap. It's only necessary to override this if you are using port mapping in the deployment spec to make the port that's visible to other pods different than the one it binds to internally. Generally, there's no good reason to do that, so this is a pretty esoteric configuration.